tests/Avalonia.RenderTests/Media/ImageBrushTests.cs

```diff
@@ -21,12 +21,18 @@ public ImageBrushTests()
 
         private string BitmapPath
         {
-            get { return System.IO.Path.Combine(OutputPath, "github_icon.png"); }
+            get
+            {
+                return System.IO.Path.Combine(OutputPath, "github_icon.png");
```

Comment:
To be clear these one-liners make sense to use `get => System.IO.Path.Combine(OutputPath, "github_icon.png");`

I was referring to the cases where there are multiple lines of code and the cases where you removed the `get` accessor entirely.